riscv-sbi.adoc

```diff
@@ -615,6 +615,53 @@ state of the hart at the time of return value verification.
 | sbi_hart_get_status      	|           2 |     0x48534D
 |===
 
+== System Reset Extension, Extension ID: 0x53525354 (SRST)
+
+The System Reset Extension provides a function that allow the supervisor
+software to request system-level reboot or shutdown.
+
+[source, C]
+----
+struct sbiret sbi_system_reset(unsigned long reset_type)
+----
+
+Reset the system based on provided reset_type. This is a synchronous call
+and does not return if it succeeds.
+
+The reset_type parameter is 32 bit wide and has following possible values:
+
+[cols="<,>",options="header,compact"]
+|===
+| Value                 	| Description
+| 0x00000000			| Shutdown
+| 0x00000001			| Cold reboot
+| 0x00000002			| Warm reboot
+| 0x00000003 - 0xEFFFFFFF	| Reserved for future use
+| 0xF0000000 - 0xFFFFFFFF	| Vendor or platform specific reset type
+| 0x100000000 - 2^XELN-1	| Reserved and unusable on RV32
```

Comment:
```suggestion
| 0x100000000 - 2^XLEN-1	| Reserved (and non-existent on RV32)
```
The description was a bit confusing, even though it's obvious when you stop and think about it what it's meant to mean. Alternatively:
```suggestion
| > 0xFFFFFFFF	| Reserved (and non-existent on RV32)
```
might be a nicer way to express the LHS.

Reply:
This suggestions looks good to me. I will update PR accordingly.